servers/visual_server.cpp

```diff
@@ -198,6 +198,15 @@ RID VisualServer::_make_test_cube() {
 	uvs.push_back(Vector3(uv_points[m_idx * 2 + 0], uv_points[m_idx * 2 + 1], 0)); \
 	vtx_idx++;
 
+#define FINAL_ADD_VTX(m_idx)                     \
```

Comment:
I think that's really overkill :)
The last increment is not an issue.

Reply:
That said nothing uses `vtx_idx`, so maybe you can just remove it altogether.

Reply:
Haha, hey, I wanna gain performance wherever I can :p